diff --git a/rpcs3/Emu/Memory/vm.cpp b/rpcs3/Emu/Memory/vm.cpp
index 639ac2ebc..be38723c3 100644
--- a/rpcs3/Emu/Memory/vm.cpp
+++ b/rpcs3/Emu/Memory/vm.cpp
@@ -410,7 +410,7 @@ namespace vm
 			{
 				if (u32 page_size = (i - start) * 4096)
 				{
-					const auto protection = start_value & page_writable ? utils::protection::rw : (start_value & page_readable ? utils::protection::ro : utils::protection::no);
+					const auto protection = start_value & page_writable || start == 0x10 ? utils::protection::rw : (start_value & page_readable ? utils::protection::ro : utils::protection::no);
 					utils::memory_protect(g_base_addr + start * 4096, page_size, protection);
 				}
 